test/mock-objects/mock-actions.js

```diff
@@ -0,0 +1,19 @@
+const { DATE: UPDATED_TIMESTAMP } = require('../mock-components/mock-dates').UPDATED
+
+module.exports = {
+  actionId: 1,
+  calculationId: 123759085,
+  calculationReference: 123759085,
```

Comment:
Why calculationId and calculationReference in same object?

Reply:
This was added because  of the schema validation .  As we are converting Id from Reference the validation I merged into this branch also had both in the mock object

Reply:
removed Reference as per the other fixes